src/mlpack/methods/ann/loss_functions/triplet_margin_loss_impl.hpp

```diff
@@ -0,0 +1,70 @@
+/**
+ * @file triplet_margin_loss_impl.hpp
```

Comment:
This should use the complete path, see https://github.com/mlpack/mlpack/blob/master/src/mlpack/methods/ann/loss_functions/dice_loss.hpp#L2 for example.